applications/ECCS/cmd/root.go

```diff
@@ -59,131 +62,146 @@ Environment Variables:
 var storeCmd = &cobra.Command{
 	Use:   "store",
 	Short: "Stores your secrets using Encryptonize",
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		return InitClient(cmd, args)
+	},
```

Comment:
You can just do this:
```suggestion
	PreRunE: InitClient,
```

Reply:
Done.